src/script/ptl-tool.py

```diff
@@ -113,6 +113,9 @@
         m = patt.match(line)
         CONTRIBUTORS[m.group(1)] = m.group(2)
 
+BZ_MATCH = re.compile("(.*https?://bugzilla.redhat.com/.*)")
+TRACKER_MATCH = re.compile("(.*https?://tracker.ceph.com/.*)")
```

Comment:
Nit: I think you could safely avoid what ever comes before `http(s)`

See below, the entire commit message is fetched.

```
jcollin@smithi042:~/test-ptl-18547$ src/script/ptl-tool.py --base master --pr-label wip-ptltool1-testing
Adding labeled PR #18284 to PR list
Will merge PRs: [18284]
Detaching HEAD onto base: master
Merging PR #18284
[ 9e4b2868 ] Ceph tracker cited: Testing ptltool commit message.http://tracker.ceph.com/issues/21393.
Checked out new branch wip-jcollin-testing-20171026
Created tag testing/wip-jcollin-testing-20171026_02

```

Reply:
We want the whole line that refers to the BZ/tracker so we can see if it says "Fixes: https://bz/" versus "Related-to: https://bz".